TutoringApp/App.js

```diff
@@ -145,6 +156,13 @@ function SearchScreen({ route }) {
   return (
     <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
       <Text>Search!</Text>
+
+      <Course CourseName="CIIC4010" Tutor={sampleScheduleData}>
```

Comment:
Try writing `<Course CourseName="CIIC4010" Tutor={sampleScheduleData}/>` to avoid extra closing calls